src/main/resources/application.yml

```diff
@@ -86,6 +86,9 @@ app:
     nqueries:
       amount: ${SECURITY_NQUERIES_AMOUNT:3}
       perminutes: ${SECURITY_NQUERIES_PERMINUTES:1}
+  privacy:
+    threshold:
+      minTotalResults: ${PRIVACY_THRESHOLD_RESULTS:30}
```

Comment:
Spring uses env vars that follow the naming of the properties by default. So I not necessarily see a reason here to add another layer of indirection.